### PR TITLE
[DC-1965] updated bucket_access_configured(bucket_name) to use StorageClient()

### PR DESCRIPTION
**Pull request by lledezma**
function utilizes IAM test permission checking instead of list access.

### CODE REVIEW

File: data_steward/tools/add_hpo.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 import bq_utils
 import constants.bq_utils as bq_consts
 import gcs_utils
```
Comment (MichaelSchmidt82) [inline data_steward/tools/add_hpo.py:19]: Is gcs_utils still being used?

Comment (lledezma) [reply data_steward/tools/add_hpo.py:19]: Good catch.

**Review by dcarbone — APPROVED** [review]
lgtm!

**Review by MichaelSchmidt82 — COMMENTED** [review]
Just a comment, not a change request.  consider longer or better descriptive variable names to improve readability.

**Pull request merged**